vunit/sim_if/ghdl.py

```diff
@@ -317,32 +321,35 @@ def _get_command(  # pylint: disable=too-many-branches
             elif self._gtkwave_fmt == "vcd":
                 sim += ["--vcd=%s" % wave_file]
 
-        if not ghdl_e:
-            cmd += sim
-            if elaborate_only:
-                cmd += ["--no-run"]
+        if not precompiled:
```

Comment:
Maybe return fast if precompiled. I.e. invert the logic of the condition and deindent the rest of the function.

Reply:
Agree!